src/lib/operators/import_binary.hpp

```diff
@@ -32,13 +32,18 @@ class ImportBinary : public AbstractReadOnlyOperator {
   /*
    * Reads the given binary file. The file must be in the following form:
    *
-   * --------------
-   * |   Header   |
-   * |------------|
-   * |   Chunks¹  |
-   * --------------
+   * -----------------
+   * |     Header    |
+   * |---------------|
+   * |     Chunks¹   |
+   * |---------------|
+   * |  Part. Header²|
+   * |---------------|
+   * |   Partitions² |
+   * -----------------
    *
    * ¹ Zero or more chunks
+   * ² Partition header is optional. If present, there is at least one partition.
```

Comment:
"at least one partition" includes the partition generated by the null partitioning, correct? Maybe this should be added for clarification.